src/components/macro/macro-action-editor/tab/macro-delay/macro-delay.component.ts

```diff
@@ -0,0 +1,34 @@
+import { EditableMacroAction } from '../../../../../config-serializer/config-items/macro-action/EditableMacroAction';
+import { AfterViewInit, Component, ElementRef, Input, OnInit, Renderer, ViewChild } from '@angular/core';
```

Comment:
Move imports from angular to the first line and add a new line after it, so it can be in a seperated group.